synapse/app/_base.py

```diff
@@ -49,7 +49,6 @@ def register_sighup(func, *args, **kwargs):
 
     Args:
         func (function): Function to be called when sent a SIGHUP signal.
-            Will be called with a single default argument, the homeserver.
```

Comment:
I decided to remove the implicit `hs` param rather than change `_reload_logging_config` to ignore it, since it seemed more confusing than helpful.